### PR TITLE
filling in blanks for 0.9.7 release docs.  Updating version,

### PR DESCRIPTION
**Pull request by albertshau**
updating screenshots, adding docs about callbacks, and various
other small fixes.

### CODE REVIEW

File: docs/docs/source/guide/admin/ui.rst [hunk 1]
```diff
@@ -436,31 +436,86 @@ editing its configurations.
...
+``Install Uses`` will not enforce the presence of the dependent service. In this example, "myapp-2.0" *uses* the "ntp" service
+at install time. This means that if the "ntp" service is also on the cluster, the installation of "ntp" will occur
+before the installation of "myapp-2.0". However, "ntp" does not have to be placed on the cluster in order for "myapp-2.0"
+to be placed on the cluster.
+
+Runtime
+-------
+Runtime dependencies contain fields that are analagous to those
+in the install section. The only difference is the service actions that they apply to. Install dependencies affect
+the INSTALL and REMOVE service actions, whereas runtime dependencies affect the INITIALIZE, START, and STOP dependencies.
```
Comment (wolf31o2) [inline docs/docs/source/guide/admin/ui.rst]: last "dependencies" should be service actions

File: docs/docs/source/guide/admin/ui.rst [hunk 2]
```diff
@@ -436,31 +436,86 @@ editing its configurations.
...
+It also means the start of "myapp-2.0" will occur after the start of "sql-db" and that the stop of "myapp-2.0"
+will occur before the stop of "sql-db". Similarly, because "myapp-2.0" *uses* "load-balancer", initialization and start of
+"myapp-2.0" will occur after the start of "load-balancer". Similarly, the stop of "myapp-2.0" will occur before the stop
+of "load-balancer". Since it is in ``uses``, enforcement of this ordering only applies if "load-balancer" is present on the
+same cluster as "myapp-2.0". The "myapp-2.0" service can be placed on a cluster without the "load-balancer" service.
+
+Provides
+--------
+The ``Provides`` field define an extra level of indirection when specifying dependencies. In this example, the "myapp-2.0"
+service provides the "myapp" service. This means that if other services can put "myapp" in their runtime or install dependencies,
```
Comment (wolf31o2) [inline docs/docs/source/guide/admin/ui.rst]: s/if//

File: docs/docs/source/guide/quickstart/index.rst [hunk 3]
```diff
@@ -141,8 +141,8 @@ contact your OpenStack administrator to get this information.
 Provisioning your First Cluster
 ===============================
 
-Click on the 'Clusters' icon on the sidebar to the left. For an administrator, this page lists all the clusters
-that have been provisioned across all Loom user accounts.
+Click on the 'Clusters' icon on the left most icon on the top bar. This page lists all the clusters
```
Comment (wolf31o2) [inline docs/docs/source/guide/quickstart/index.rst]: It's the right most icon

**Comment by wolf31o2** [comment]
A few comments, otherwise LGTM

File: docs/docs/source/guide/admin/ui.rst (hunk 1 above)
Comment (albertshau) [inline docs/docs/source/guide/admin/ui.rst]: good catch. done.

File: docs/docs/source/guide/admin/ui.rst (hunk 2 above)
Comment (albertshau) [inline docs/docs/source/guide/admin/ui.rst]: done

File: docs/docs/source/guide/quickstart/index.rst (hunk 3 above)
Comment (albertshau) [inline docs/docs/source/guide/quickstart/index.rst]: done

**Pull request merged**